perf-tool/src/server.cpp

```diff
@@ -75,7 +75,7 @@ void Server::handleServer()
     serverSocketFd = socket(AF_INET, SOCK_STREAM, 0);
     if (serverSocketFd < 0)
     {
-        error("ERROR opening socket");
+        perror("ERROR opening socket");
```

Comment:
If the socket cannot be opened there is no point in continuing with `bind`, `listen` etc